indico/modules/events/editing/client/js/components/EditingView/MenuBar.jsx

```diff
@@ -0,0 +1,71 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2020 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+import managementURL from 'indico-url:event_management.settings';
+import displayURL from 'indico-url:events.display';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Header, Icon} from 'semantic-ui-react';
+import {Translate} from 'indico/react/i18n';
+
+import './MenuBar.module.scss';
+
+export default function MenuBar({eventId, menuItems}) {
+  return (
+    <div styleName="menu-bar">
+      <Header as="h2" styleName="header">
+        <Translate>Paper Editing</Translate>
+      </Header>
+      <ul styleName="list">
+        <li>
+          <Translate>OTHER MODULES</Translate>
+          <ul styleName="inner-list">
+            {menuItems.map(item => (
+              <a key={item.name} href={item.url}>
+                <li styleName="inner-list-item">{item.title}</li>
+              </a>
+            ))}
+          </ul>
+        </li>
+      </ul>
+      <ul styleName="list">
+        <li>
+          <Translate>OTHER VIEWS</Translate>
```

Comment:
I'd write this normally and [use CSS](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform) to capitalize it.